scrapers/tx/events.py

```diff
@@ -9,23 +9,44 @@
 
 import pytz
 
+bill_re = re.compile(r"(SJR|HCR|HB|HR|SCR|SB|HJR|SR) (\d+)")
+
 
 class TXEventScraper(Scraper, LXMLMixin):
     _tz = pytz.timezone("US/Central")
 
+    events_seen = set()
+
+    # Checks if an event is a duplicate.
+    # Events are considered duplicate if they have the same
+    # name, date, start time, and end time
+    def is_duplicate(self, event):
```

Comment:
This is a good symptom-treating approach for preventing the duplicates. Were able you able to diagnose what was causing the duplicates in the first place, or has that remained a mystery so far?